usesCleartextTraffic information (#1050)
apache/cordova-docs: www/docs/en/dev/guide/platforms/android/index.md

    </body>
</html>
```
### Android Quirks
The default API level in the Cordova Android platform has been upgraded. On an Android 9 device, clear text communication is now disabled by default.

> Indicates whether the app intends to use cleartext network traffic, such as cleartext HTTP. The default value for apps that target API level 27 or lower is "true". Apps that target API level 28 or higher default to "false".

Reviewer: ```suggestion

```
This might be confusing in this context. Better link to the Android docs below.

> When the attribute is set to "false", for example, HTTP and FTP will refuse the app’s requests to use cleartext traffic. The key reason for avoiding cleartext traffic is the lack of confidentiality, authenticity, and protections against tampering; a network attacker can eavesdrop on transmitted data and also modify it without being detected.

Reviewer: ```suggestion
By default HTTP and FTP etc. will refuse the apps requests to use cleartext traffic. The key reason for avoiding cleartext traffic is the lack of confidentiality, authenticity, and protections against tampering; a network attacker can eavesdrop on transmitted data and also modify it without being detected. You can learn more about the `android:usesCleartextTraffic` setting in the [documentation for Android developers](https://developer.android.com/guide/topics/manifest/application-element).
```

Sorry for the next suggestions. I always find something while proofreading a second time. I want to make sure this setting is properly explained  and attributed with it's security implications.

Reviewer: Is it better to link the Android docs here? Open for opinions.

> When the attribute is set to "false", platform components (for example, HTTP and FTP stacks, DownloadManager, and MediaPlayer) will refuse the app’s requests to use cleartext traffic. Third-party libraries are strongly encouraged to honor this setting as well. The key reason for avoiding cleartext traffic is the lack of confidentiality, authenticity, and protections against tampering; a network attacker can eavesdrop on transmitted data and also modify it without being detected.

Reviewer: ```suggestion
When the attribute is set to "false", for example, HTTP and FTP will refuse the app’s requests to use cleartext traffic. The key reason for avoiding cleartext traffic is the lack of confidentiality, authenticity, and protections against tampering; a network attacker can eavesdrop on transmitted data and also modify it without being detected.
```

I thinkt this would make it easier for Cordova app developers to understand.

Author: looks good to me.

    </body>
</html>
```
### Android Quirks
The default API level in the Cordova Android platform has been upgraded. On an Android 9 device, clear text communication is now disabled by default.

By default HTTP and FTP etc. will refuse the apps requests to use cleartext traffic. The key reason for avoiding cleartext traffic is the lack of confidentiality, authenticity, and protections against tampering; a network attacker can eavesdrop on transmitted data and also modify it without being detected. You can learn more about the `android:usesCleartextTraffic` or any other android application elements setting in the [documentation for Android developers](https://developer.android.com/guide/topics/manifest/application-element).

To allow clear text communication again, set the `android:usesCleartextTraffic` on your application tag to true in `config.xml` file:
`<platform name="android">
  <edit-config file="app/src/main/AndroidManifest.xml" mode="merge" target="/manifest/application">
      <application android:usesCleartextTraffic="true" />
  </edit-config>

> </platform>`

Reviewer 2: I think this is intended to be a preformatted block
```suggestion
```xml
<platform name="android">
  <edit-config file="app/src/main/AndroidManifest.xml" mode="merge" target="/manifest/application">
      <application android:usesCleartextTraffic="true" />
  </edit-config>
</platform>
```
```

    </body>
</html>
```
### Android Quirks
The default API level in the Cordova Android platform has been upgraded. On an Android 9 device, clear text communication is now disabled by default.

By default HTTP and FTP etc. will refuse the apps requests to use cleartext traffic. The key reason for avoiding cleartext traffic is the lack of confidentiality, authenticity, and protections against tampering; a network attacker can eavesdrop on transmitted data and also modify it without being detected. You can learn more about the `android:usesCleartextTraffic` or any other android application elements setting in the [documentation for Android developers](https://developer.android.com/guide/topics/manifest/application-element).

To allow clear text communication again, set the `android:usesCleartextTraffic` on your application tag to true in `config.xml` file:
```xml
<platform name="android">
  <edit-config file="app/src/main/AndroidManifest.xml" mode="merge" target="/manifest/application">
      <application android:usesCleartextTraffic="true" />
  </edit-config>
</platform>

> 

Reviewer 2: Oops, maybe I messed up in my last suggestion, but the block end is missing after the `</platform>`

We should have a ``` after `<platform>`
